kubespawner/spawner.py

```diff
@@ -433,6 +442,15 @@ def _hub_connect_port_default(self):
         """
     )
 
+    user_storage_match_labels = Dict(
+        help="""
```

Comment:
It should probably also support expanding {username} and friends in the label values, to provide more targeted matching. Let's definitely get it out of this PR?